composer.json

```diff
@@ -13,8 +13,10 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^11.0",
-    "pimcore/admin-ui-classic-bundle": "^1.1"
+    "php": "~8.1.0 || ~8.2.0",
```

Comment:
is this necessary?
```suggestion
```

Reply:
I think we should set supported php versions. But is a different topic yes.

Reply:
ok, but let's keep it in there, since i am also adding the "retroactive" changelog